bootstrap/scripts/2-download.sh

```diff
@@ -46,7 +46,7 @@ if [ ! -e "${BOOTSTRAP_DOWNLOADS}/vmBootstrap/pharo" ]; then
 	mkdir ${BOOTSTRAP_DOWNLOADS}/vmBootstrap
 	cd ${BOOTSTRAP_DOWNLOADS}/vmBootstrap
 
-	${BOOTSTRAP_REPOSITORY}/bootstrap/scripts/getPharoVM.sh 100 vm $BOOTSTRAP_ARCH
+	${BOOTSTRAP_REPOSITORY}/bootstrap/scripts/getPharoVM.sh 120 vm $BOOTSTRAP_ARCH
```

Comment:
All changes are good to me but this one.
The image initializing the boostrap is still a P10 image. We need to invest some times to update this but it is not something fast. So we need to keep P10 here IIRC

Reply:
Also, IIRC using the P12 VM works but is slower because it is not adapted to the image running the bootstrap.